app/uk/gov/hmrc/customs/notification/repo/NotificationWorkItemRepo.scala

```diff
@@ -45,6 +45,8 @@ trait NotificationWorkItemRepo {
   def blockedCount(clientId: ClientId): Future[Int]
 
   def deleteBlocked(clientId: ClientId): Future[Int]
+
+  def unblock(): Future[Int]
 }
 
```

Comment:
I think you need a compound index on 1) `status` 2) `updatedAt` to support the query selector in your new update command.